modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -135,7 +135,18 @@ const EditorUpload = function (editor: Editor): EditorUpload {
             blobCache.removeByUri(image.src);
             replaceImageUriInView(image, uploadInfo.url);
           } else if (uploadInfo.error) {
-            ErrorReporter.uploadError(editor, uploadInfo.error);
+            if (uploadInfo.error.options.remove) {
+              if (editor.hasPlugin('rtc')) {
```

Comment:
We have `Rtc.isRtc(editor)` as a central check for this 👍 

I didn't realise it was a dynamic option returned by the handler, this is certainly easier than overriding the entire upload handler in the plugin.

Reply:
We seem to use isRtc and hasPlugin an equal amount of times ( three ) in the code. Is there a notable difference between the two?

Reply:
`Rtc.isRtc()` is only available in core and is the preferred method. The `hasPlugin` variant is needed for other plugins, as it isn't part of core.

As a side note, I didn't consider forking here, but that does seem like a better solution for now. Nice work!

Reply:
Worth noting that the existing `hasRtc` check could be improved to use the new hasPlugin API. eg:
```
export const isRtc = (editor: Editor) => editor.hasPlugin('rtc', true);
```